src/Microsoft.AspNet.Server.Testing/Deployers/SelfHostDeployer.cs

```diff
@@ -50,11 +51,11 @@ private CancellationToken StartSelfHost()
             var commandName = DeploymentParameters.Command;
             if (string.IsNullOrEmpty(commandName))
             {
-                commandName = DeploymentParameters.ServerType == ServerType.WebListener ? "web" : "kestrel";
+                commandName = DeploymentParameters.ServerType == ServerType.WebListener ? "weblistener" : "kestrel";
```

Comment:
FYI: The standard command names have changed to:
'web' = Kestrel
'weblistener = WebListener